packages/roosterjs-react-command-bar/lib/plugins/RoosterCommandBarPlugin.Shortcuts.ts

```diff
@@ -0,0 +1,179 @@
+        shiftKey: false,
+        which: KeyCodes.z,
+        command: RoosterCommandBarCommands.Undo
+    },
+    // Redo for Mac: Command (meta) + SHIFT + Z
+    {
+        metaKey: true,
+        ctrlKey: false,
+        shiftKey: true,
+        which: KeyCodes.z,
+        command: RoosterCommandBarCommands.Redo
+    },
+    // Bullet for Mac: Command (meta) + .
+    {
+        metaKey: true,
+        ctrlKey: false,
+        shiftKey: false,
+        which: KeyCodes.period,
+        command: RoosterCommandBarCommands.Bullet
+    },
+    // Numbering for Mac: Command (meta) + /
+    {
+        metaKey: true,
+        ctrlKey: false,
+        shiftKey: false,
+        which: KeyCodes.forwardSlash,
+        command: RoosterCommandBarCommands.Numbering
+    },
+    // Insert link for Mac: Command (meta) + k
+    {
+        metaKey: true,
+        ctrlKey: false,
+        shiftKey: false,
+        which: KeyCodes.k,
+        command: RoosterCommandBarCommands.InsertLink
+    }
+];
+
+const winCommands: ShortcutCommand[] = [
+    // Bold for Windows: Ctrl + B
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.b,
+        command: RoosterCommandBarCommands.Bold
+    },
+    // Italic for Windows: Ctrl + I
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.i,
+        command: RoosterCommandBarCommands.Italic
+    },
+    // Underline for Windows: Ctrl + U
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.u,
+        command: RoosterCommandBarCommands.Underline
+    },
+    // Undo for Windows: Ctrl + Z
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.z,
+        command: RoosterCommandBarCommands.Undo
+    },
+    // Redo for Windows: Ctrl + Y
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.y,
+        command: RoosterCommandBarCommands.Redo
+    },
+    // Bullet for Windows: Ctrl + .
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.period,
+        command: RoosterCommandBarCommands.Bullet
+    },
+    // Numbering for Windows: Ctrl + /
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.forwardSlash,
+        command: RoosterCommandBarCommands.Numbering
+    },
+    // Insert link for Windows: Ctrl + k
+    {
+        metaKey: false,
+        ctrlKey: true,
+        shiftKey: false,
+        which: KeyCodes.k,
+        command: RoosterCommandBarCommands.InsertLink
+    }
+];
+
+export function getCommandFromEvent(event: PluginEvent): RoosterCommandBarCommands {
+    if (event.eventType !== PluginEventType.KeyDown) {
+        return RoosterCommandBarCommands.None;
+    }
+
+    const commands = browserData.isMac ? macCommands : winCommands;
```

Comment:
Interesting way to handle meta and ctrl keys :)

Reply:
it's actually refactored from DefaultShortcut; ctrl+k link adding was actually missing mac support